sites/website/src/docs/resources/browser-support.md

```diff
@@ -6,7 +6,7 @@ custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src
 ---
 The following browsers have native support for the Web Components features used by `fast-element` and our components:
 
-* Edge 79+
+* Microsoft Edge 79+
 * Firefox 63+
 * Chrome 67+
 * Safari 10.1+
```

Comment:
Should these also be prefixed with company names?
```suggestion
* Mozilla Firefox 63+
* Google Chrome 67+
* Apple Safari 10.1+
```

Reply:
If ours must, it doesn't hurt anything to do so for the others for better consistency.